lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -0,0 +1,60 @@
+{
+  "workflows": [
+    {
+      "name": "default",
+      "label": "Digital collections workflow",
+      "description": "A multi-step workflow for digital collections",
+      "actions": [
+         {
+           "name": "finalize_digitization",
```

Comment:
Switching the workflow to be MD-centered is a separate issue. This is just to wire up notifications.